### DISCUSSION TITLE
显示sprite的控件

### DISCUSSION THREAD

**Discussion by BestP2P** [netease-im/NIM_Duilib_Framework#306] [General]
你好，请问如果我一个控件，只是一个sprite，就是显示一个图片而已，应该用什么控件好呢？为什么不弄一个sprite？我应该用Control来达到目的吗？

**Comment by nmgwddj** [comment]
Control 控件 bkimage 可以达到你的需求，你可以基于 Control 创建一个派生类来实现更多图片相关功能，你可以随意给它命名。
